ivy/functional/backends/paddle/experimental/losses.py

```diff
@@ -64,3 +64,31 @@ def smooth_l1_loss(
     return paddle.nn.functional.smooth_l1_loss(
         input, target, reduction=reduction, beta=beta
     )
+
+
+@with_unsupported_device_and_dtypes(
+    {
+        "2.5.1 and below": {
+            "cpu": (
+                "bfloat16float16",
```

Comment:
You are missing a break line here, should look like this:
"bfloat16",
"float16",